src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleThreadContext.java

```diff
@@ -121,12 +127,25 @@ List<ModuleExtensionUsageBuilder> getExtensionUsageBuilders() {
   }
 
   static class ModuleExtensionUsageBuilder {
+
+    private record RepoOverride(
+        String exportedName,
```

Comment:
these names should probably be updated to match the semantics of `override_repo`. So `exportedName` should be `overriddenRepoName`, and `localRepoName` should be `overridingRepoName`. same elsewhere